worlds/tloz/docs/en_The Legend of Zelda.md

```diff
@@ -0,0 +1,43 @@
+# The Legend of Zelda (NES)
+
+## Where is the settings page?
+
+The [player settings page for this game](../player-settings) contains all the options you need to configure and export a
+config file.
+
+## What does randomization do to this game?
+
+All acquirable pickups (except maps and compasses) are shuffled amongst each other. Logic is in place to ensure both
```

Comment:
```suggestion
All acquirable pickups (except maps and compasses) are shuffled among each other. Logic is in place to ensure both
```
This isn't necessary by any means, but I prefer among to amongst when trying to write accessible material.